modules/gcloud/bigquery_test.go

```diff
@@ -81,8 +87,74 @@ func ExampleRunBigQueryContainer() {
 		}
 	}
 
-	fmt.Println(val)
-
+	fmt.Println(val[0])
 	// Output:
-	// [30]
+	// 30
+}
+
+func TestBigQueryWithDataYamlFile(t *testing.T) {
+	ctx := context.Background()
+
+	testDataPath, err := filepath.Abs(filepath.Join(".", "testdata"))
+	require.NoError(t, err)
+
+	r, err := os.Open(filepath.Join(testDataPath, "data.yaml"))
+	require.NoError(t, err)
+
+	bigQueryContainer, err := gcloud.RunBigQuery(
+		ctx,
+		"ghcr.io/goccy/bigquery-emulator:0.6.1",
+		gcloud.WithProjectID("test"),
+		gcloud.WithDataYAML(r),
+	)
```

Comment:
suggestion: use dataYaml for this to simplify.
```suggestion
	bigQueryContainer, err := gcloud.RunBigQuery(
		ctx,
		"ghcr.io/goccy/bigquery-emulator:0.6.1",
		gcloud.WithProjectID("test"),
		gcloud.WithDataYAML(bytes.NewReader(dataYaml)),
	)
```